internal/tools/tools.go

```diff
@@ -27,4 +27,6 @@ import (
 	_ "golang.org/x/exp/cmd/apidiff"
 	_ "golang.org/x/tools/cmd/goimports"
 	_ "golang.org/x/vuln/cmd/govulncheck"
+
+	_ "go.opentelemetry.io/collector/internal/tools/semconvkit"
```

Comment:
Should this live in go-build-tools?

Reply:
Maybe, but I didn't want to block on this. I'm happy to open a follow up issue

Reply:
Ya followup thing